colearn_pytorch/pytorch_learner.py

```diff
@@ -30,11 +31,27 @@
 import torch.utils.data
 from torch.nn.modules.loss import _Loss
 
+from opacus import PrivacyEngine
+
 from colearn.ml_interface import MachineLearningInterface, Weights, ProposedWeights
 
 _DEFAULT_DEVICE = torch.device("cpu")
 
 
+@dataclass
+class DiffPrivConfig:
+    target_epsilon: float
+    target_delta: float
+    max_grand_norm: float
```

Comment:
misspelled grad

Reply:
this is implemented in ml_interface. please check the keras implementation for reference and use `from colearn.ml_interface import DiffPrivBudget, DiffPrivConfig, TrainingSummary, ErrorCodes`